docs/docs/widgets/star-rating.md

```diff
@@ -26,7 +26,7 @@ The **Star Rating** component can be used to display as well as input ratings. T
 
 | <div style={{ width:"100px"}}> Event </div> | <div style={{ width:"100px"}}> Description </div> |
 |:----------- |:----------- |
-| On Change | This event is triggered when a star is clicked. |
+| On change |  Triggers whenever the user clicked a star |
```

Comment:
Hey, can you please check the sentence in the description column? I think it is grammatically incorrect.